components/constellation/constellation.rs

```diff
@@ -1195,6 +1217,16 @@ where
                 self.forward_event(destination_pipeline_id, event);
             },
             FromCompositorMsg::SetCursor(cursor) => self.handle_set_cursor_msg(cursor),
+            FromCompositorMsg::EnableProfiler(rate) => {
+                for chan in &self.sampling_profiler_control {
+                    let _ = chan.send(SamplerControlMsg::Enable(rate));
```

Comment:
Perhaps `expect` or `warn` on failure for both sends to the monitor? The constellation keeps two channels to the monitor so something is wrong if the monitor isn't receiving messages anymore...